__tests__/MSW_node_test.res

```diff
@@ -32,10 +33,33 @@ describe("Node Server", () => {
       |> then_(text => expect(text) |> toEqual("jihchi/res-msw") |> resolve)
     )
 
+    testPromise("post with request body works", () => {
+      let payload = Js.Dict.fromArray([
+        ("owner", Js.Json.string("jihchi")),
+        ("repo", Js.Json.string("res-msw")),
+      ])
+
+      Fetch.fetchWithInit(
+        "https://api.github.com/repos",
+        Fetch.RequestInit.make(
+          ~method_=Post,
+          ~body=Fetch.BodyInit.make(Js.Json.stringify(Js.Json.object_(payload))),
+          ~headers=Fetch.HeadersInit.make({"Content-Type": "application/json"}),
+          (),
+        ),
+      )
+      |> then_(Fetch.Response.text)
+      |> then_(text => expect(text) |> toEqual("jihchi/res-msw") |> resolve)
+    })
+
     testPromise("post works", () =>
       Fetch.fetchWithInit(
         "https://api.github.com/repos/jihchi/res-msw",
-        Fetch.RequestInit.make(~method_=Post, ()),
+        Fetch.RequestInit.make(
+          ~method_=Post,
+          ~headers=Fetch.HeadersInit.make({"Content-Type": "application/json"}),
+          (),
+        ),
```

Comment:
**question**: Do we still need this?

Reply:
I simplified this to just a regular Post, thanks!